index.d.ts

```diff
@@ -18,4 +18,23 @@ for await (const chunk of stream) {
 }
 ```
 */
-export default function mergeStreams(streams: NodeJS.ReadableStream[]): NodeJS.ReadableStream;
+export default function mergeStreams(streams: NodeJS.ReadableStream[]): MergedStreams;
+
+/**
+Single stream combining the output of multiple streams.
+*/
+export type MergedStreams = NodeJS.ReadableStream & {
+	/**
+	Pipe a new readable stream.
+
+	This fails if `MergedStream` has already ended.
```

Comment:
Maybe be specific on how it fails? Does the method throw, does any of the streams (input or merged) emit error event, if so, which stream.

Reply:
Fixed in 0a2089adad0b276900505b86d0e888df3425344e.